### PR TITLE
fix: adding more resources in the lua script

### PR DESCRIPTION
**Pull request by edgarsilva948**
### What does this PR do?

adjusts the terraform bootstrap to have the new script added to the eksctl here: #198 

### Motivation

#198 

### More

- [ ] Yes, I have tested the PR using my local account setup  (Provide any test evidence report under Additional Notes)

- [ ] Yes, I have added a new example under [examples](https://github.com/aws-samples/crossplane-aws-blueprints/tree/main/examples) to support my PR

- [ ] Yes, I have updated the [docs](https://github.com/aws-samples/crossplane-aws-blueprints/tree/main/doc) for this feature

- [ ] Yes, I have linked to an issue or feature request (applicable to PRs that solves a bug or a feature request)

**Note**:
 - Not all the PRs require examples and docs
 - We prefer small, well tested pull requests. Please ensure your pull requests are self-contained, and commits are squashed

### For Moderators
- [ ] E2E Test successfully complete before merge?

### CODE REVIEW

File: bootstrap/terraform/values/argocd.yaml [hunk 1]
```diff
@@ -110,14 +110,42 @@ configs:
           end
 
           return health_status
-      %{ if crossplane_upjet_aws_provider_enable }
```
Comment (candonov) [inline bootstrap/terraform/values/argocd.yaml:113]: why removing this?

File: bootstrap/terraform/values/argocd.yaml [hunk 2]
```diff
@@ -149,15 +177,46 @@ configs:
           end
 
           return health_status
-      %{endif}
-      %{ if crossplane_aws_provider_enable }
```
Comment (candonov) [inline bootstrap/terraform/values/argocd.yaml:153]: and this?

**Review by candonov — COMMENTED** [review]
why remove the conditionals?

File: bootstrap/terraform/values/argocd.yaml (hunk 2 above)
Comment (edgarsilva948) [reply bootstrap/terraform/values/argocd.yaml]: fixed

File: bootstrap/terraform/values/argocd.yaml (hunk 1 above)
Comment (edgarsilva948) [reply bootstrap/terraform/values/argocd.yaml]: fixed

File: bootstrap/terraform/values/argocd.yaml [hunk 3]
```diff
@@ -158,6 +187,39 @@ configs:
...
+              if v == val then
+                return true
+              end
+            end
+            return false
+          end
+
+          local has_no_status = {
+            "Composition",
+            "CompositionRevision",
```
Comment (candonov) [inline bootstrap/terraform/values/argocd.yaml:201]: are compositions and compositionrevisions `*.apiextensions.crossplane.io`, not *.aws.crossplane.io`?

File: bootstrap/terraform/values/argocd.yaml [hunk 4]
```diff
@@ -158,6 +187,39 @@ configs:
...
+            "DeploymentRuntimeConfig",
+            "ControllerConfig",
```
Comment (candonov) [inline bootstrap/terraform/values/argocd.yaml:203]: Same question `deploymentruntimeconfigs.pkg.crossplane.io` and `controllerconfigs.pkg.crossplane.io` not `*.aws.crossplane.io`?

File: bootstrap/terraform/values/argocd.yaml [hunk 5]
```diff
@@ -198,6 +260,110 @@ configs:
...
```
Comment (candonov) [inline bootstrap/terraform/values/argocd.yaml:276]: same question here?

File: bootstrap/terraform/values/argocd.yaml [hunk 6]
```diff
@@ -198,6 +260,110 @@ configs:
...
```
Comment (candonov) [inline bootstrap/terraform/values/argocd.yaml:347]: and here

**Comment by edgarsilva948** [comment]
Lua script adjusted to match the crossplane [doc](https://docs.crossplane.io/latest/guides/crossplane-with-argo-cd/).

File: bootstrap/terraform/values/argocd.yaml [hunk 7]
```diff
@@ -110,55 +110,40 @@ configs:
           end
 
           return health_status
-      %{ if crossplane_upjet_aws_provider_enable }
-      "*.aws.upbound.io/*":
+
+      "*.upbound.io/*":
```
Comment (candonov) [inline bootstrap/terraform/values/argocd.yaml:114]: There is already "*.upbound.io/*" higher up in this file.

Comment (edgarsilva948) [reply bootstrap/terraform/values/argocd.yaml:114]: we have one for *.upbound.io/*, another one for awsblueprints.io/* and another for *.crossplane.io/*.

**Review by candonov — APPROVED** [review]
Nice one!

**Pull request merged**